examples/nodejs/lambda-examples/cloudwatch-metrics/docker/ecs-code/index.ts

```diff
@@ -0,0 +1,77 @@
+import {GetSecretValueCommand, SecretsManagerClient} from '@aws-sdk/client-secrets-manager';
+import {
+  CacheClient,
+  Configurations,
+  DefaultMomentoLoggerFactory,
+  DefaultMomentoLoggerLevel,
+  CredentialProvider,
+} from '@gomomento/sdk';
+import {ExperimentalMetricsLoggingMiddleware} from '@gomomento/sdk/dist/src/config/middleware/experimental-metrics-logging-middleware';
+
+const _secretsClient = new SecretsManagerClient({});
+const _cachedSecrets = new Map<string, string>();
+let _cacheClient: CacheClient | undefined = undefined;
+
+async function main() {
+  try {
+    const loggerFactory = new DefaultMomentoLoggerFactory(DefaultMomentoLoggerLevel.DEBUG);
+    const logger = loggerFactory.getLogger('CloudWatchMetricsMiddlewaresExample');
+
+    const cacheClient = await getCacheClient(loggerFactory);
+    logger.info('Created a CacheClient configured with metrics middleware');
+
+    logger.info('Issuing 5 minutes of set and get requests to generate data for the dashboard example');
+    for (let i = 0; i < 60 /* seconds */ * 5 /* minutes */; i++) {
+      await cacheClient.set('cache', 'metrics-example-{i}', 'VALUE');
+      await cacheClient.get('cache', 'metrics-example-{i}');
+      await delay(1000);
+    }
+
+    logger.info('Momento metrics middleware example complete!');
+  } catch (err) {
+    console.log('Error occurred in the ECS example code:', err);
+  }
+}
+
+async function getCacheClient(loggerFactory: DefaultMomentoLoggerFactory): Promise<CacheClient> {
+  const apiKeySecretName = process.env.MOMENTO_API_KEY_SECRET_NAME;
+  if (apiKeySecretName === undefined) {
+    throw new Error("Missing required env var 'MOMENTO_API_KEY_SECRET_NAME");
+  }
+  if (_cacheClient === undefined) {
+    const momentoApiKey = await getSecret(apiKeySecretName);
+
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-assignment
```

Comment:
just a nit but curious why this eslint hint is necessary here?

Reply:
can't recall why anymore, but I've removed the hints